sdk/lib/rtl/heapuser.c

```diff
@@ -127,3 +127,32 @@ RtlInitializeHeapManager(VOID)
     RtlInitializeCriticalSection(&RtlpProcessHeapsListLock);
 }
 
+ULONG NTAPI
+RtlGetProcessHeaps(_In_ ULONG count, _Out_cap_(HeapCount) HANDLE *heaps)
```

Comment:
Suggestion: use the parameter names already in place in the header:
https://git.reactos.org/?p=reactos.git;a=blob;f=sdk/include/ndk/rtlfuncs.h;hb=e7f6b473e6b2fb28efac42ee9cabb21db9171a73#l942
```suggestion
ULONG
NTAPI
RtlGetProcessHeaps(
    _In_ ULONG HeapCount,
    _Out_cap_(HeapCount) HANDLE *HeapArray)
```

Reply:
already pushed that, but indeed.